webapp/packages/plugin-sql-editor-new/src/SQLEditor/SQLCodeEditorPanel/SQLCodeEditorPanel.tsx

```diff
@@ -24,16 +24,38 @@ import { useSqlDialectAutocompletion } from '../useSqlDialectAutocompletion';
 import { useSqlDialectExtension } from '../useSqlDialectExtension';
 import style from './SQLCodeEditorPanel.m.css';
 import { useSQLCodeEditorPanel } from './useSQLCodeEditorPanel';
+import { EditorView } from '@codemirror/view';
 
+type ILocalSQLCodeEditorPanelState = {
+  selection: { from: number; to: number };
+};
```

Comment:
it's better to use `interface` instead or remove `I` prefix from the name

Reply:
Replaced it with an `interface`.